docs/user-guide/secrets/index.md

```diff
@@ -700,7 +700,7 @@ make that key begin with a dot.  For example, when the following secret is mount
       {
         "name": "dotfile-test-container",
         "image": "gcr.io/google_containers/busybox",
-        "command": "ls -l /etc/secret-volume",
+        "command": [ "ls -l /etc/secret-volume" ],
```

Comment:
change this to `[ "ls", "-l", "/etc/secret-volume" ],`, while this will work in this case, it won't work in all cases